Symfony/CS/Fixer/Contrib/AlignEqualsFixer.php

```diff
@@ -24,49 +24,64 @@ class AlignEqualsFixer extends AbstractAlignFixer
     /**
      * {@inheritdoc}
      */
-    public function fix(\SplFileInfo $file, $content)
+    public function fix(\SplFileInfo $file, Tokens $tokens)
     {
-        list($tokens, $deepestLevel) = $this->injectAlignmentPlaceholders($content);
+        $this->deepestLevel = 0;
```

Comment:
That is a matter of state-less fixers and lack of `setUp` and `tearDown` methods. Already some of the fixer take advantage of state-awareness,
Can't normalize it the other way - second fixer need a state for it's recursive way of working, and it wouldn't be simpler (in matter of more readability) for it to change it into state-less version.